.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,12 @@
-PR Checklist:
+## Description
+<! -- What is it meant to do? -->
 
-- [ ] add a short description of what's changed to the top of the `CHANGELOG.md`
-- [ ] add/update tests (or don't, for reasons explained below)
 
-## Describe this PR
+## Linked Issues
+<! -- Use a key word (e.g. closes or resolves) to close related issues  -->
+
+
+## Tests and performance
+
+ - [ ] Changes to any .js files are covered by existing tests or this PR adds new tests (if not please explain why below)
+ - [ ] No new packages are added or package size and performance considerations are discussed below
```

Comment:
perhaps also add something to the effect of "- [ ] PR title fits our _change log format_". Then we can link that to a public notion doc or something that walks through the possible prefixes (BugFix:, Feature:, etc), title describes work clearly, is appropriate and safe for community, etc

Reply:
Oh I like it!